samples/bluetooth/central_past/src/main.c

```diff
@@ -293,7 +295,7 @@ int main(void)
 		sync_create_param.options = 0;
 		sync_create_param.sid = per_sid;
 		sync_create_param.skip = 0;
-		sync_create_param.timeout = 0xaa;
+		sync_create_param.timeout = per_adv_interval_ms * 10 / 10; /* 10 attempts */
```

Comment:
typo?

```suggestion
		sync_create_param.timeout = per_adv_interval_ms * 10; /* 10 attempts */
```

Reply:
Rather it's a missing comment. 

`timeout` is in 10s of milliseconds, so a timeout value of `10` means `100` milliseconds, thus the division of 10 from ms to 10s of ms.

Reply:
ah, i see it now. Yeah, comment should be better worded maybe, as this is confusing IMO. Maybe just drop the "10 / 10" altogether and make the comment say "10 attempts (timeout is expressed in 10s of ms)" or something like that?
Anyway, I'm glad I made this a non-blocking comment :)

Reply:
Maybe we should add a macro BT_GAP_MS_TO_PER_ADV_TIMEOUT to make the code self-documenting?

Reply:
In the LE audio samples and test we something like this
```c
static uint16_t interval_to_sync_timeout(uint16_t interval)
{
	uint32_t interval_ms;
	uint32_t timeout;

	/* Add retries and convert to unit in 10's of ms */
	interval_ms = BT_GAP_PER_ADV_INTERVAL_TO_MS(interval);
	timeout = (interval_ms * PA_SYNC_INTERVAL_TO_TIMEOUT_RATIO) / 10;

	/* Enforce restraints */
	timeout = CLAMP(timeout, BT_GAP_PER_ADV_MIN_TIMEOUT, BT_GAP_PER_ADV_MAX_TIMEOUT);

	return (uint16_t)timeout;
}
```